src/main/groovy/de/dkfz/roddy/execution/jobs/cluster/slurm/SlurmJobManager.groovy

```diff
@@ -26,13 +27,13 @@ class SlurmJobManager extends GridEngineBasedJobManager {
     private final ZoneId TIME_ZONE_ID
 
     private final Map<String, JobState> stateMap = [
-            "RUNNING": JobState.RUNNING,
-            "SUSPENDED": JobState.SUSPENDED,
-            "PENDING": JobState.HOLD,
+            "RUNNING"   : JobState.RUNNING,
+            "SUSPENDED" : JobState.SUSPENDED,
+            "PENDING"   : JobState.HOLD,
```

Comment:
Is this `+` in `CANCELLED` OK, or is this just a typo?

Reply:
that is what i saw on the cluster, but in the doku it was without the +

Sadly i cant check currently should i just add both?

Reply:
That would be an option that does not hurt, except for the extra code.